pylint/checkers/imports.py

```diff
@@ -850,6 +866,83 @@ def _check_imports_order(
                         )
         return std_imports, external_imports, local_imports
 
+    def _get_out_of_order_string(
+        self,
+        third_party_imports: list[tuple[ImportNode, str]] | None,
+        first_party_imports: list[tuple[ImportNode, str]] | None,
+        local_imports: list[tuple[ImportNode, str]] | None,
+    ) -> str:
+        # construct the string listing out of order imports used in the message
+        # for wrong-import-order
+        third_party_str = (
+            "third party import(s) "
```

Comment:
Nit: I would remove the _str suffixes from the variable name in this function.

Reply:
Resolved in commit 6146c03